src/worker/lambda/zygote/multiTree.go

```diff
@@ -14,6 +14,10 @@ type MultiTree struct {
 	trees []*ImportCache
 }
 
+func (mt *MultiTree) Warmup() error {
+	panic("multi-tree warmup not implemented!")
```

Comment:
I generally like to avoid panics because we won't do cleanup then (e.g., maybe something we mounted never gets unmounted).  Let's just return an error.